src/pages/assign-calls.js

```diff
@@ -32,7 +32,11 @@ export default function AssignCallsPage() {
     useEffect(async () => {
         const authorisedCallTypesGateway = new AuthorisedCallTypesGateway();
         let authCallTypes = await authorisedCallTypesGateway.getCallTypes();
-        setFilteredCallTypes(authCallTypes);
+        let callNames = [];
+        for (const type in authCallTypes) {
+            callNames.push(authCallTypes[type].name);
+        }
+        setFilteredCallTypes(callNames);
```

Comment:
35 - 39 could be 1 line:

setFilteredCallTypes(authCallTypes.map(callType => callType.name));

Same for other changes in the commit.